memcached-operator/test/e2e/memcached_test.go

```diff
@@ -51,7 +51,7 @@ func TestMemcached(t *testing.T) {
 	})
 }
 
-func memcachedScaleTest(t *testing.T, f *framework.Framework, ctx framework.TestCtx) error {
+func memcachedScaleTest(t *testing.T, f *framework.Framework, ctx *framework.TestCtx) error {
```

Comment:
if the `ctx` here `ctx := framework.NewTestCtx(t)` is a pointer. The user is mostly likely to pass it to other place as a pointer which can force the intended behavior.

Reply:
I made the change in the sdk with this PR: https://github.com/operator-framework/operator-sdk/pull/451

After that PR is merged, I will update this repo with the new changes